pkg/component/ai/instill/v0/main.go

```diff
@@ -101,12 +102,46 @@ func (e *execution) Execute(ctx context.Context, jobs []*base.Job) error {
 		defer gRPCCientConn.Close()
 	}
 
+	var result []*structpb.Struct
+
+	// We will refactor the component soon to align the data structure with Instill Model.
+	// For now, we move out `TASK_EMBEDDING` to a separate section.
```

Comment:
👍 Thanks for documenting this, it helps understand the code when you're visiting the package and you're not familiar with it